scripts/ci/transifex.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex
+
+tx_install() {
+    pip install transifex-client
+}
+
+tx_init() {
+    echo "[https://www.transifex.com]
+hostname = https://www.transifex.com
+username = ${TRANSIFEX_USER}
+password = ${TRANSIFEX_PASSWORD}
+" > ~/.transifexrc
+}
+
+tx_push() {
+    tx push --source --parallel
+}
+
+echo "Publishing to Transifex..."
+tx_install
+tx_init
+tx_push
```

Comment:
not sure if you need functions vs just inlining

Reply:
yea, meant to move these to inline